docs/config/README.md

```diff
@@ -107,7 +107,7 @@ A conditional format string wrapped in `(` and `)` will not render if all variab
 
 For example:
 
-- `(@$region)` will show nothing if the variable `region` is `None`, otherwise `@` followed by the value of region.
+- `(@$region)` will show nothing if the variable `region` is `None` or empty string, otherwise `@` followed by the value of region.
 - `(some text)` will always show nothing since there are no variables wrapped in the braces.
 - When `$all` is a shortcut for `\[$a$b\] `, `($all)` will show nothing only if `$a` and `$b` are both `None`.
```

Comment:
Should this also be updated to the following? 
```suggestion
 - When `$all` is a shortcut for `\[$a$b\] `, `($all)` will show nothing only if `$a` and `$b` are both `None` or empty strings.
```

Reply:
I've create such test and it fails so I think no. Should it also handle that case? I don't quite understand `map_meta` for now
```rust
    #[test]
    fn test_empty_meta_variable() {
        const FORMAT_STR: &str = "$all";
        const FORMAT_STR__ALL: &str = "$a$b";

        let formatter = StringFormatter::new(FORMAT_STR)
            .unwrap()
            .map_meta(|var, _| match var {
                "all" => Some(FORMAT_STR__ALL),
                _ => None,
            })
            .map(|var| match var {
                "a" => Some(Ok("")),
                "b" => Some(Ok("")),
                _ => None,
            });
        let result = formatter.parse(None).unwrap();
        assert_eq!(result.len(), 0);
    }
```

Reply:
I think we should probably support this but maybe we can do it later in a separate PR.